concepts/observability-strategy/configurable-monitoring/04-monitoring-api.md

```diff
@@ -7,89 +7,94 @@ apiVersion: telemetry.kyma-project.io/v1alpha1
 metadata:
   name: PrometheusRemoteWrite
 spec:
-  receiver: # singular, different receivers need different pipelines and exporterssss configs
-    type: application # application | system | custom, default is application
-    otlp: # maps to central tail pipeline, dealing with the actual application logs
-      namespaces: [] # generates the rule for the rewrite_tag assigned to every pipeline
-      excludeNamespaces: []
-      containers: []
-      excludeContainers: []
+  receiver: # singular, different receivers need different pipelines and exporters configs
```

Comment:
Not sure what you mean with "pipelines and exporters configs":

- different configs for pipeline and exporter

```suggestion
  receiver: # singular, different receivers need different pipeline and exporter configs
```

- or different pipelines, and different exporter configs

```suggestion
  receiver: # singular, different receivers need different pipelines and exporter configs
```